.buildkite/packaging.pipeline.yml

```
@@ -120,10 +120,9 @@ steps:
           - x-pack/packetbeat
           - x-pack/winlogbeat
 
-      - label: "SNAPSHOT: {{matrix}} docker Linux/arm64"
+      - label: "SNAPSHOT: {{matrix}} Linux/arm64 and Darwin/arm64"
```

`nit`: there is no `Darwin/arm64` support for `7.17`

```suggestion
      - label: "SNAPSHOT: {{matrix}} Linux/arm64"
```